lib/rspec_api_documentation/writers/formatter.rb

```diff
@@ -3,9 +3,9 @@ module Writers
     module Formatter
 
       def self.to_json(object)
-        JSON.pretty_generate(object.as_json)
+        JSON.generate(object.as_json)
```

Comment:
This isn't totally desirable - it looks like pretty_generate was a deliberate choice to make diffs of the json nicer.

I believe the actual issue here is this one: https://github.com/flori/json/pull/195 - but see also https://github.com/flori/json/issues/181 and https://github.com/jruby/jruby/issues/1609